odoo/addons/base/tests/test_float.py

```diff
@@ -145,6 +145,7 @@ def try_round(amount, expected, precision_rounding=None):
         try_round(457.3, '455.00', precision_rounding=5)
         try_round(457.5, '460.00', precision_rounding=5)
         try_round(457.1, '456.00', precision_rounding=3)
+        try_round(2.5, '2.50', precision_rounding=0.05, method='DOWN')
```

Comment:
@nim-odoo It would be good to have a test with negative values as well, and possibly replicate the tests we added for `UP` at 7705f883d23b71cdb1d30300ce69d9df665ae55e, which is to use both exact values (such as 2.50) and values that are slightly biased in the wrong direction (for DOWN this would be values slightly below, such as 2.40)

@smetl @qdp-odoo Next time you need to change our float utils, please be sure to get the patch reviewed by multiple people, and extend the float_utils unit tests with an appropriate set of test values, exercising the various corner cases. This is absolutely mandatory!
Something as obvious as overlooking the sign of the epsilon correction should have been detected very early.